### DISCUSSION TITLE
PLEASE READ BEFORE POSTING A DISCUSSION

### DISCUSSION THREAD

**Discussion by murphy-john** [Anura3D/Anura3D_OpenSource#29] [Announcements]
Before posting a discussion topic:

1. Search the discussion threads to see if your question has already been answered.
2. Use the [debug feature](https://docs.microsoft.com/en-us/visualstudio/debugger/debugging-absolute-beginners?view=vs-2022&tabs=csharp) in Visual Studio to locate the source of the issue in the code. 
3. Provide enough detail about the problem you're trying to model with Anura3D. Explain the features you're using. 
4. Review the tutorial manual for instructions on getting started and example problems.

**Comment by 1229524533** [comment]
May I know if Anura3D community will offer any training course for beginners in 2023? If yes, please share it's details.

**Reply by murphy-john** [reply]
Hello Israr, we will be sending out an announcement for our next Workshop coming up in June. The announcement will be out within the next week.

**Comment by 1229524533** [comment]
Thank you John for your reply. I wish the workshop is online and can be designed and planned such that beginners can also get benefited. And also it can explain the steps involved in installing the Anura3D software because I am personally facing the installation problem. My A3D folder is not giving me proper output files. The calculate file is not .bat and two files are missing inside. I installed visual studio and the required toolkit but still facing the same problem. So, now I cannot procyany further in my analysis.&nbsp;


Dr. Israr
Postdoctoral Fellow
China



---Original---
From: "John ***@***.***&gt;
Date: Fri, Apr 28, 2023 21:37 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [Anura3D/Anura3D_OpenSource] PLEASE READ BEFORE POSTING ADISCUSSION (Discussion #29)




 
Hello Israr, we will be sending out an announcement for our next Workshop coming up in June. The announcement will be out within the next week.

**Reply by ayerrocolom** [reply]
Dear Israr,

The online workshop is tomorrow online. Please, find the program and the free registration link here [https://github.com/Anura3D/Anura3D_OpenSource/wiki/Anura3D-Online-Workshop-2023-&-official-release](url)

Best,
Alba

**Comment by girmameaza12** [comment]
i use Anura 3D for  rainfall induced landslide analysis and i follow the video and the manual tutorial ,but in the calculation parts it shows the following error, and for (audiometric compression test) example ,the calculation done without error.can you please give me solution.
![Uploading image.png…]()

**Comment by 199918xzl** [comment]
Who has the 2022 version of Anura code?  Can you share a copy? thanks!
![178bac50f7cef0e094fe3054254e857](https://github.com/Anura3D/Anura3D_OpenSource/assets/61869153/08b32902-b719-4870-bcc6-0ff8fb386182)

**Reply by murphy-john** [reply]
If you go in the "src" folder, there is a "history" button on the right above the folders and files listed. From there, you can find the histories of the revisions with time. Go to the point before the 2023 release on June 30th, it looks like the Commits on March 3rd are what you want, and select the "<>" button. Pressing this button will take you to the state of the repository at that point, which should be the 2022 version. From there, you can download the code as explained in the tutorial manual and videos. 

I'm sure there are more elegant ways to do this using the command line or the Github Desktop app but the way I've explained has worked for me in the past.

**Reply by 199918xzl** [reply]
Thank you very much, I found it!

**Comment by Hosseinan9675** [comment]
Hi, 
I tried to Setup the Anura3D, I have downloaded the latest version the Anura 3D code, and I could successfully compile the code on the Visual Studio 2022. I don't have  any idea for the next step "Located the Executable "! What Should I do? Also, I could not see the Anura3D.exe file on the downloaded Zip file.

**Reply by XJ-liu-9527** [reply]
see Anura3D TutorialManual_2022/2023,2 Getting started with Anura3D
![image](https://github.com/user-attachments/assets/9b618df9-7944-45cd-880d-24c51ab8f34a)

**Reply by Hosseinan9675** [reply]
Thanks, Would you please, provide me the Link for the TutorialManual_2022/2023? unfortunately I couldn't find the file!

**Reply by XJ-liu-9527** [reply]
https://github.com/Anura3D/Anura3D_OpenSource/tree/main/manuals
first: 
![image](https://github.com/user-attachments/assets/a8a5bdc6-256c-4666-be29-a07137d9f5e9)
second:
![image](https://github.com/user-attachments/assets/f97eb9d7-0b12-4a8a-8f66-529564fe1ee7)

**Reply by Hosseinan9675** [reply]
Thank you very much! I followed the recommendations to Setup the Anura3D and It is done!

**Comment by XJ-liu-9527** [comment]
Hi!
I am a beginner about ANURA3D. Now I have a problem. Suppose I set two "Remove Fixities" in the x direction, let's say the names are RF1 and RF2. How exactly can I delete RF1 from a CPS file? Currently, modifying the CPS file will remove all X directions by running the following command: $$REMOVE_FIXITIES 1 0 0.
Thanks a lot.
